bin/test-run

```diff
@@ -35,6 +35,30 @@ function run_test(){
     go test -v "$1" --linkerd="$linkerd_path" --linkerd-namespace="$linkerd_namespace" --k8s-context="$k8s_context" --integration-tests "$2"
 }
 
+function install_edge() {
+  set -x
+  tmp=$(mktemp -d -t l5dbin.XXX)
+  trap "rm -rf $tmp" EXIT
+
+  curl https://run.linkerd.io/install-edge | HOME=$tmp sh
+
+  local linkerd_path=$tmp/.linkerd2/bin/linkerd
+  $linkerd_path install --proxy-auto-inject --linkerd-namespace="$linkerd_namespace" | kubectl apply -f - > /dev/null 2>&1
+  $linkerd_path check --linkerd-namespace="$linkerd_namespace" > /dev/null 2>&1
+  set +x
+}
+
+function run_upgrade_test() {
+  local linkerd_namespace="$linkerd_namespace"-upgrade
```

Comment:
my bash foo is not great, does this mean that `$linkerd_namespace` becomes something like `l5d-integration-upgrade` when calling into `install_edge()`? if so, i'd prefer we use a different var name, or just pass a namespace in, that way it's clear reading through the script that `$linkerd_namespace` always means the same thing.